report_github_metrics.rb

```diff
@@ -18,10 +18,14 @@ def collect_job_metrics(job, tags)
   ]
 end
 
-def collect_workflow_metrics(jobs, tags)
+def collect_workflow_metrics(jobs, tags, branch)
   start = jobs.min_by{|job| job["started_at"]}["started_at"]
   finish = jobs.max_by{|job| job["completed_at"]}["completed_at"]
   status = jobs.all?{|job| ["success", "skipped"].include? job["conclusion"]} ? "success" : "failure"
+  if !ENV['BRANCH'].nil? && ENV['BRANCH'] != ''
```

Comment:
Do we want to execute this in the default `[]` case?

My assumption is no... that if you aren't specifying tagged branches, then no metric should have the `branch` tag. But I believe in this code in such a case all metrics will have `branch:other` tagged if no tagged-branches is set.

What do you think? Am I misunderstanding the code?

Reply:
Good point 👍 Updated.